tik_manager4/core/filelog.py

```diff
@@ -2,26 +2,18 @@
 import os
 import datetime
 
-
 class Filelog(object):
+    # FIXME(ckutlu): We should definitely rethink the need for global state as
```

Comment:
I personally don't like this too but couldn't find a better solution to pass the messages to ui at this point.

Reply:
Also, if you dont like this wait until you take a peek at objects.guard  :)

Reply:
Yeah, we definitely should not have any global state.  There are instances where it may be necessary, e.g. you are shipping a library and you wanna make certain things convenient for users of that library.